spec/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
+  # (RE)XML related helper methods.
+  module XML
+    # Converts `REXML::Attributes` into a `Hash`.
+    #
+    # @param [REXML::Attributes] attributes The attributes to convert
+    #   into a Hash.
+    # @return [Hash] the attribute keys mapped to the attribute values
+    def attributes_to_hash(attributes)
+      Hash[attributes.to_enum.to_a]
```

Comment:
I preferred chaining `Enumerable#to_h` to `REXML::Attributes#to_enum` but that isn't as portable (not supported in some earlier Ruby versions).